content/05.heterogeneity.md

```diff
@@ -1,3 +1,20 @@
 ### Techniques to manage disparities in data generation are required to power robust analyses in rare diseases
->Rarity of patients leads to heterogeneity in sample collection, causing disparities in the data. 
->We will discuss how rigorous normalization and methodologies capturing sample-wise gene-set level information can help appropriate integration of disparate data points to power machine learning approaches11–13. 
+
+As with common diseases, genomic and transcriptomic data from rare diseases can suffer from artifacts introduced by batch, processing methodology, sequencing platform, or other non-biological phenomena. 
+This effect is amplified in rare diseases which often have few samples and heterogenous phenotypes.
+Furthermore, because datasets are many times pieced together from multiple small studies, disease phenotype or other important biological characteristics are often confounded by the previously mentioned "batch" factors. 
+A key consideration here is, if possible, active dialogue with the data generators or experts in the field who may have unexpected insights into potential sources of variation. 
+One example of the value of this, experienced by the authors, occurred when studying tumors associated with the disease neurofibromatosis type 1. 
+These datasets were, unbeknownst to the computational biologists, generated from samples obtained with vastly different surgical techniques (laser ablation and excision vs standard excision), resulting in substantial biological differences that are a consequence of process, not reality. 
+Consequently, careful assessment of confounding factors and implementation of normalization methods is important to identifying biologically meaningful features within a dataset. 
+Assessment of confounding factors and heterogeneity in rare disease datasets is perhaps most easily performed using unsupervised learning approaches such as clustering and dimensionality reduction. 
+Clustering methods like k-means clustering or hierarchical clustering can be used to characterize the structure present in many different types of data such as genomic, imaging, and health records data to identify batch effects. 
+Similarly, a variety of dimensionality reduction methods are can be used to visualize sample heterogeneity and potential confounding variables, including multidimensional scaling (MDS), principal components analysis (PCA), t-distributed stochastic neighbor embedding (tSNE), and uniform manifold approximation and projection (UMAP), among many others. [@doi:10.1007/978-3-540-33037-0_14; @doi:10.1098/rsta.2015.0202; @https://lvdmaaten.github.io/publications/papers/JMLR_2008.pdf; @arXiv:1802.03426]
+All of these methods can be effectively used to identify batch effects and other structure in the data (cite), though some, like tSNE and UMAP, have parameters that can substantially affect the output, and thus the interpretation, of the analysis [@doi:10.23915/distill.00002;@arXiv:1802.03426]. 
```

Comment:
an example of the parameters referred to here can be helpful in understanding how they might affect the output.

Reply:
My thought was that the citations would be sufficient here, but i'll add a mention of perplexity/nearest neighbors. I think that `@doi:10.23915/distill.00002` gets this point across better than i ever could!